lib/forms/item_types.js

```diff
@@ -119,16 +119,21 @@ function processInputSchema(comp, schema) {
 }
 
 function setComponentOptions(comp, options, setModelFunc) {
+    function trySetModelFunc(comp, data) {
+        if (! comp.isDestroyed()) setModelFunc(comp, data);
+    };
```

Comment:
Don't need the semi colon on the end of a function definition.

Reply:
Also is it a convention that you follow where you put a space between `!` and the variable? I've never seen it before outside of a few places in CC. What's the reasoning for it, or is it just a preference?

Reply:
1) - Oops, no idea how he snuck in there.  I removed
2) Can't say I've ever given it much thought.  :)